src/components/KSegmentedControl/KSegmentedControl.vue

```diff
@@ -90,48 +95,54 @@ export default defineComponent({
 @import '@/styles/variables';
 @import '@/styles/functions';
 
-.segmented-control .k-button {
-  --KButtonPrimaryBase: var(--blue-100);
-  --KButtonPrimaryHover: var(--blue-100);
-  color: var(--KSegementedControlPrimary, var(--blue-500, color(blue-500)));
-  border-radius: 0;
-  margin-left: -1px;
-  flex: 1;
-
-  &.primary {
-    z-index: 1;
-    border-color: var(--blue-500);
-  }
+.segmented-control {
+  column-gap: var(--KSegmentGap, 0px);
+
+  .k-button {
```

Comment:
Since this is within `scoped` styles, don't you need `:deep(.k-button) {` for this selector?

Reply:
Added, but it looks like the styles somehow work without it which makes no sense...